src/Utils/ClassFinder.php

```diff
@@ -82,7 +82,7 @@ protected static function findClasses(array $paths): array
                 $classname = $namespace . $name;
                 if (class_exists($classname)) {
                     if (isset($result[$name])) {
-                        throw new RuntimeException('Duplicate Model "' . $name . '": '. $result[$name] . ' + ' . $classname);
+                        throw new RuntimeException('Duplicate class "' . $name . '": '. $result[$name] . ' + ' . $classname);
```

Comment:
fehler in `QueryType.php` fängt kleingeschrieben an
`throw new RuntimeException('no resolver found for '. $info->returnType->toString());`
trivial :sweat_smile: